docs/get-started/installation/custom-hardware/installation/iso.md

```diff
@@ -4,7 +4,10 @@ sidebar_position: 2
 
 # ISO Installation
 
-When you want to perform a clean installation by installing the base operating system and DAppNode. Installing from an ISO will wipe the machine data and install Debian + DAppNode.
+Installing from the official DAppNode ISO will wipe the existing machine data and replace it with a Debian operating system and DAppNode. 
+
+# ISO Installation on an Intel NUC
+If you are installing DAppNode on an Intel NUC, please ensure you have upgraded your BIOS to the latest version. Instructions for downloading and installing these updates can be found on the manufacturers website.
```

Comment:
```suggestion
If you are installing DAppNode on an Intel NUC 10+ that was not purchased through DAppNode, please ensure you have upgraded your BIOS to the latest version.  All units we ship out are upgraded and confirmed to work with the current software. Instructions for downloading and installing these updates can be found on the manufacturer's website [here](https://www.intel.com/content/www/us/en/support/articles/000005636/intel-nuc.html) or [here](https://www.intel.com/content/www/us/en/support/articles/000033291/intel-nuc.html) depending on model.  All downloads of firmware, BIOS files, and software can be found on both above links.
```

Thanks for your contribution, but in the future please open an issue first making sure it's not duplicated, before making a PR